azurelinuxagent/common/protocol/wire.py

```diff
@@ -111,22 +109,6 @@ def get_certs(self):
         certificates = self.client.get_certs()
         return certificates.cert_list
 
-    def get_vmagent_manifests(self):
```

Comment:
WireClient maintains a reference to the goal state (returned by get_goal_state). WireClient is held within a WireProtocol, which is returned by ProtocolUtil, which is a singleton. Basically, this makes the goal state a global variable, and it has introduced threading issues in the past. We have been removing the use of WireClient.get_goal_state() over time. This PR presented an opportunity to remove a couple of them.